test_1:
  initial_pos_ugv_x: -9.4
  initial_pos_ugv_y: -2.2
  initial_pos_ugv_z: 0.0
  initial_pos_ugv_yaw:  0.0
  initial_pos_ugv_pitch: 0.0
  initial_pos_ugv_roll: 0.0

